src/main/java/net/webtide/tools/github/User.java

```diff
@@ -83,4 +83,13 @@ public boolean isSiteAdmin()
     {
         return permissions;
     }
+
+    @Override
+    public String toString()
+    {
+        return "User{" + "login='" + login + '\'' + ", type='" + type + '\'' + ", name='" + name + '\'' + ", company='"
+            + company + '\'' + ", location='" + location + '\'' + ", email='" + email + '\'' + ", bio='" + bio + '\''
+            + ", siteAdmin=" + siteAdmin + ", permissions=" + permissions + ", createdAt=" + createdAt + ", updatedAt="
```

Comment:
I would rather see this toString be smarter, and not include attributes that are null/unset in it's output